cabal-testsuite/PackageTests/Freeze/T9799b/src/None.hs

```diff
@@ -0,0 +1,10 @@
+{-# LANGUAGE TemplateHaskell #-}
+module None where
```

Comment:
I don't recall :=). This module should build because cabal freeze with `setup.libA == 0.1.0.0` will generate a freeze file with `any.libA == 0.1.0.0 | == 0.2.0.0`, so version `0.2.0.0` will be picked for both Setup and for this module (meaning this module succeeds to compile, but the Setup will fail)